configs/edison/config.20161006bugfix.alpha8.A_WCYCL1850S.ne30_oEC_ICG.edison

```diff
@@ -131,6 +131,7 @@ polarPlot = False
 [streamfunctionMOC]
 ## options related to plotting the streamfunction of the meridional overturning 
 ## circulation (MOC)
+maxChunkSize = 1000
```

Comment:
ok, thanks for clarifying, I had forgotten that we still have the global value in the input section. But I am still confused by one thing: I seem to understand that, when an analysis task does not specify a maxChunkSize then chunking is set to `None`, whereas `chunking=maxChunkSize` occurs when an analysis task does specify a maxChunkSize value. So, when would the case `chunking=True' occur?